AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_component_validation.py

```diff
@@ -215,3 +214,46 @@ def validate_asset_property(property_name: str, get_asset_value: typing.Callable
     assert expected_asset.id.is_equal(expected_asset.id, set_value), \
         f"Error: The {component_name}'s {property_name} property failed to properly set the mesh. Asset Id: " \
         f"{expected_asset.id} was expected but Asset Id: {set_value} was retrieved from \"{expected_asset.get_path()}\""
+
+
+def validate_script_canvas_graph_file(get_script_canvas_graph_file: typing.Callable,
+                                      set_script_canvas_graph_file: typing.Callable, sc_file_path: str) -> None:
+    """
+    Function to validate the setting of a script canvas graph file to the file source field in the script canvas component
+
+    get_script_canvas_component_graph_file: getter for the script canvas component file source handle field
+    set_script_canvas_component_graph_file: setter for the script canvas component's file source field.
+    sc_file_path: the path on disk where the graph file exists
+
+    """
+    Report.info(f"Validating Script Canvas component's file source field can be set.")
+
+    old_value = get_script_canvas_graph_file()
+
+    set_script_canvas_graph_file(sc_file_path)
+
+    set_value = get_script_canvas_graph_file()
+    assert set_value != old_value and set_value is not None, f"Graph file could not be set!"
```

Comment:
Add some parentheses around the two conditions to make it clear to a reader skimming through what the intent of the test conditions are.

Reply:
good call. parentheses added